### ISSUE TITLE
Remove sensitive info

### ISSUE THREAD

**Issue by coool9** [Bilal-Nizamani/Middleware#1]
You have exposed your password and someone's mobile number when doing mongodb credentials

**Comment by Bilal-Nizamani** [comment]
Yes I will remove it  the database is in my local machine won't be affected by anyone
Thanks by the way.
